packages/synapse-react-client/src/components/SynapseHomepageV2/SynapseHomepageV2.tsx

```diff
@@ -0,0 +1,514 @@
+import React from 'react'
+import {
+  Box,
+  Button,
+  SxProps,
+  Typography,
+  Chip,
+  useTheme,
+  useMediaQuery,
+  Link,
+} from '@mui/material'
+import { TypeAnimation } from 'react-type-animation'
+import { ReactComponent as Image1 } from '../../assets/homepage/image1.svg'
+import { SynapseTrendingProjects } from './SynapseTrendingProjects'
+import { SynapseByTheNumbers } from './SynapseByTheNumbers'
+import { SynapseFeatures } from './SynapseFeatures'
+import { SynapseInAction } from './SynapseInAction'
+import { backgroundInlineJpeg } from '../../assets/homepage/header-splash'
+import { SynapsePartners } from './SynapsePartners'
+import { SynapsePlans } from './SynapsePlans'
+import { useInView } from 'react-intersection-observer'
+import SageFullLogo from '../../assets/icons/SageFullLogo'
+import { SynapseFeaturedDatasets } from './SynapseFeaturedDatasets'
+import { onSearch, SynapseHomepageSearch } from './SynapseHomepageSearch'
+import { SynapseHomepageNavBar } from './SynapseHomepageNavBar'
+
+const synapseInActionTable = 'syn61670075'
+const past30DaysDownloadMetricsTable = 'syn61597084'
+const generalStatsMetricsTable = 'syn61588163'
+const featuredDatasetsTable = 'syn61609402'
+const searchAutocompleteTable = 'syn61670515'
+
+const popularSearches = [
+  "Alzheimer's Disease",
+  'Parkinson',
+  'Neurofibromatosis',
+  'HTAN',
+  'ukb-ppp',
+  'ROSMAP',
+  'GENIE',
+]
+
+export const darkTextColor = '#22252A'
+export const homepageBodyText: SxProps = {
+  fontSize: '24px',
+  fontWeight: 400,
+  lineHeight: '34px',
+  color: darkTextColor,
+}
+
+const h2Sx: SxProps = {
+  fontSize: {
+    xs: '36px',
+    md: '52px',
+  },
+  fontWeight: 600,
+  lineHeight: '52px',
+  color: darkTextColor,
+}
+
+const defaultHomepageText: SxProps = {
+  color: darkTextColor,
+  fontWeight: 600,
+}
+const titleSx: SxProps = {
+  ...defaultHomepageText,
+  fontWeight: 300,
+  fontSize: {
+    xs: '48px',
+    md: '72px',
+  },
+  lineHeight: {
+    xs: '120%',
+    md: '82px',
+  },
+  color: 'white',
+}
+const sidePadding: SxProps = {
+  pl: {
+    xs: '15px',
+    sm: '50px',
+  },
+  pr: {
+    xs: '15px',
+    sm: '50px',
+  },
+}
+
+export const SynapseHomepageV2: React.FunctionComponent = () => {
+  const theme = useTheme()
+  const isDesktopView = useMediaQuery(theme.breakpoints.up('lg'))
+
+  //optimization - prioritize loading above-the-fold content (delay loading below the fold)
+  const { ref, inView } = useInView({ triggerOnce: true })
+  return (
+    <Box>
+      <SynapseHomepageNavBar />
+      <Box sx={{ position: 'relative', pb: '80px' }}>
+        <Box
```

Comment:
For a lot of these boxes that just use `clipPath` to render some visual element, you could extract them into tiny custom components or just JSX constants, where the component/constant name may help identify what visual element the component corresponds to.

E.g. in TableColumnSchemaForm there are constants like `HIERARCHY_VERTICAL_LINE_COMPONENT`, `HIERARCHY_END_COMPONENT` that fulfill similar purposes.